.github/workflows/docker-push.yaml

```diff
@@ -15,16 +15,16 @@ jobs:
     name: docker-push
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/setup-go@v2
+    - uses: actions/checkout@v4
       with:
-        go-version: '1.18'
+        fetch-depth: 0
 
-    - uses: actions/checkout@v2
+    - uses: actions/setup-go@v5
       with:
-        fetch-depth: 0
+          go-version-file: go.mod
 
     - name: Login to Quay
-      uses: docker/login-action@v1
+      uses: docker/login-action@v3
       with:
         registry: quay.io
         username: ${{ secrets.QUAY_USERNAME }}
```

Comment:
Is there a reason for the step order change?

Reply:
Same as https://github.com/red-hat-storage/ocs-client-operator/pull/92#discussion_r1512926724